### PR TITLE
Add best practices for sharing data..

### PR DESCRIPTION
**Pull request by dkhamsing**

### CODE REVIEW

**Comment by tiagoalmeida** [comment]
We knew this would happen some day :(. You need to rebase.

**Comment by dkhamsing** [comment]
:smile:

File: Issues/Week75.md [hunk 1]
```diff
@@ -47,3 +47,4 @@
 
 * [UXPin Free Resources](http://www.uxpin.com/knowledge.html), by [@uxpin](https://twitter.com/uxpin)
 * [Use React Native](http://www.reactnative.com/)
+* [WWDC 2015 announced: June 8-12](http://www.macrumors.com/2015/04/14/wwdc-2015-june-8-12/)
```
Comment (RuiAAPeres) [inline Issues/Week75.md]: Please remove this

**Comment by dkhamsing** [comment]
uhhh... hang on

**Comment by dkhamsing** [comment]
yea will do

**Comment by dkhamsing** [comment]
Updated

**Pull request merged**